### DISCUSSION TITLE
Vision and Purpose Manifesto

### DISCUSSION THREAD

**Discussion by seekanddefine** [knowledge-for-good/knowledgeforgood#10] [Show and tell]
*WIP*

Dear humans,

Knowledge for Good is committed to empowering humans for a more balanced future by providing tools for digital literacy/scholarship, frameworks, and evidence-based strategies that develop whole people to become stronger life-long learning practitioners, more confident knowledge gatherers/digital humanists, and skilled gardeners of mind and body. By incrementally and iteratively increasing the developmental outcomes of students through intentional, educational micro-structures that address neurocognitive factors (eg. space repetition) to build skills, we can personalize student support while simultaneously making success more accessible. Better educated people take better care of themselves and others and it is our essential nature to heal, grow, and protect. Success doesn't come from being intelligent but with passion (limiting beliefs are a self-fulfilling prophecy) , persistence (willingness to experience discomfort welcomes change), and grit (willingness to make mistakes and fail forward).

We believe that individualized learner goals are best met with simple, high-quality technology (technology is a tool, but not a destination) built on actionable data and cooperative learner feedback. ‘There is no failure. Only feedback.’–Robert Allen Places of ignorance are best navigated by embracing humanity as individuals and as a collective whole that address essential needs collaboratively. This allows us to build universal capacity within a welcoming, magnetic, inclusive, fun community where people cohesively operate with agency and belonging. We believe in what we are doing, are devoted to helping others, and know are part of something bigger than can have an impact on the sustainability of the planet and the human species. We are committed to achieving the highest good for all and are compelled to make a meaningful and lasting impact to make the world a better place for all living things. 

We believe this is the most important work we can do and pursue it with reckless abandon.

‘Far and away the best prize that life offers is the chance to work hard at work worth doing.’ —Theodore Roosevelt

We start by asking: Who am I becoming? Who are you becoming? Who are we becoming? What can we become by learning together?

The following are truths are held with a lifetime commitment: 

We believe our greatest investment is in people.

We are here to make something useful together.

We are here to intentionally create a safe, stable, healthy, and durable space for learning.

We believe that a life can be hard and terrible, but we have us and our ability to learn.

We believe existence = coexistence. 

We believe ethical engagement as citizens within a global community is mandatory and that seeking understanding of interdependence encourages personal and collective flourishing. 

![image](https://user-images.githubusercontent.com/38668794/122988152-09890900-d367-11eb-9a0a-3c4a9d4c2668.png)
https://educationoftheheartdialogue.org/educating-for-a-shared-humanity/

We believe intentional community fosters strong coping skills. 

We believe connectedness in practice is the best route forward. 

We believe we can see a better way forward by thinking together and accept empathy as a mandatory tool to recognize that someone else's journey is just as real as our own. 

We design experiences to stretch and grow people to find success. Satisfaction comes with attuning not attainment. 

We connect the dots by relating and relying on each other, finding meaning in unpredictability and uncertainty, and establishing comfort and stability through mindful awareness. Sense and Sense Stability. 

We believe that faith in the links among all things unites humankind, that knowledge bridges connections between people and that contagious enthusiasm allows us to stay on the cutting edge of knowledge by sharing what we are learning.

We believe in expanding traditional notions of social and emotional learning to incorporate the training of attentional and emotional skills, compassion and altruism, and systems thinking.

We believe people are raw potential and thoughts are raw material.

We believe that everyone can learn (neuroplasticity) and ask not "Can you learn?" but "How can you learn best?"

We believe deep learning is the new electricity.

We believe that how well we learn determines how well we engage with the world and our larger purpose. 

We believe that the best learners are teachers.  ‘Teaching is the highest form of understanding.’ –Aristotle

We believe that humanity is self-assembled and stronger together by sharing resources (open-source). 

We believe participants are free to subscribe or cancel at any time.

We invite all to join who believe others can grow in the ethical reliability, truthfulness, capability, and strength of others when we work together for good. 

We believe we are better learners when surrounded by other good learners. ‘When one teaches, two learn.’ –Robert Heinlein

We believe using strategic tools that are energy-waste neutral to organize human knowledge and building collective learner context support better decision-making.

We believe we can harness the power of self-propelled, perpetuating tools and scaffolding to create circumstances where humans produce their best ideas.

In other words we believe doing good reduces harm, increases well-being, improves the public good resources available.

We believe appropriate action enhances people's embodied metaphorical construal of abstract concepts.

Our work is fueled by the individual growth and wellbeing of our community citizens and as such do everything possible to cultivate potential and facilitate growth by noticing where we are growing, tracking our individual and mutual learning paths, and celebrating each other through help and encouragement. By recognizing and cultivating the potential in others we use the evidence of progress and our data to design solutions for small improvements which unlock our capacity to change, grow, and develop for the better.

‘Teaching is only demonstrating that it is possible. Learning is making it possible for yourself.’ –Paulo Coelho

We aim to break down knowledge silos and believe that the right to education is a basic human right: knowledge starvation should be a prosecutable crime. Education=individual and universal freedom. We develop autonomous capabilities to unlock possibilities previously unknown and shed illusions through acquiring knowledge, practicing wisdom, and driving individual philosophical inquiry and cultural participation. Rich diversity creates better products. Understanding the lived experiences of humans across time, age. ethnicity, family, distance, race, religion, economic level, language, and culture reveals human commonalities. Enjoying the company of people who have face and conquered life's difficulties allows access to wisdom that we lack. Connection facilitates connecting disparate phenomena and defying conventional thinking to move outside the box. We revel in taking in the world we know and using dimensional thinking to view problems from new angles embracing the strange and enlightening to stimulate the dreams of tomorrow.

We believe this is best done by facilitating radical democracy (knowledge congress/learning congress) where every self-directed learner can build from where they are at through the power of radical compassion.

“We must develop a feeling of dearness to the entirety of sentient beings… For someone who really takes the idea of universal compassion seriously, then any sentient being that is deserving of compassion can never be an object deserving of our wrath, anger, or hatred. There cannot be such a thing as an enemy, because anyone who is deserving of compassion should be seen as a true friend.”

The health of the republic depends on the health of it's citizens. By recognizing and celebrating individuality we overcome our distance and strangeness from one another. 

We believe cognitive dissonance and willful ignorance are best addressed through test-driven learning path cultivation and are audaciously committed to the intrinsic unification of knowledge.

We deepest core values are: compassion (non-violence/non-harming/non-exploiting), accessibility, connectedness (collaboration/coming together to face life's mysteries, no individual is fully formed in isolation), respect...

We believe violence is preventable.

We believe compassion builds endurance. 

We believe progressive practice creates self-sufficiency which in turn creates global sustainability. 

We strive to be clear and careful about the manufacturing of kool-aid. 

We strive to create a culture of inquiry that finds deep joy in paying attention, expanding horizons to secure opportunity and confidence in the future. 

“So, friends, every day do something that won’t compute…Give your approval to all you cannot understand…Ask the questions that have no answers. Put your faith in two inches of humus that will build under the trees every thousand years…Laugh. Be joyful though you have considered all the facts….Practice resurrection.”–Wendell Berry

We believe we can do things better than we have in the past and recognize the importance of acknowledging what our ancestors have fought for and are committed to documenting our progress openly in order to create a rich, transparent community of practice that serves all participants. We embrace collaborative storytelling and document oral history with care. We strive to complete the lineage of the thought leaders, educators, and researchers' shoulders we stand on. 

We believe innovative thinking can foster dialogue to reveal trends, raise issues, identify opportunities, and offer solutions. 

Our process:

1. Set intention: I believe I can learn.
2. Cultivate potential through deconstruction to discover constituent parts and internal assumptions (cognitive biases): In the past, I believed I wasn't smart enough which limited what I could learn.
3. Examine and study (develop bench strength, notice where others have come before): I have an idea. I see these problems and solutions. I recognize specific mechanisms and processes involved. I see how they are connected to my plan and goals.
4. Create and innovate: I scope alternate ways to proceed knowing my first idea may not be the best. 
5. Choose best option. I clarify and improve with comparison and contrast. 
6. Fully reflect: I see patterns and related issues in coordination with my progress.
7. Future generation (ensure success): I see these potential repercussions and plan responses. (Forethought helps navigate obstacles before they appear.)

We celebrate our achievements, reward outstanding contributions, and push to invigorate bodymind awareness and address social injustices through the lens of intersectionality. 

We are learners without borders. 

Lastly, we believe being part of another person's development is one of the best experiences in the world and core to our own individual growth as learners. 

‘A teacher affects eternity: he can never tell where his influence stops.’ –Henry Adams

**Comment by avidrucker** [comment]
I love this! I would like to be able to effectively discuss individual points. On that note, I will quote a few pieces that I am curious to know more about.

**Comment by avidrucker** [comment]
> more confident knowledge gatherers

What are effective ways of gathering knowledge, and for what purposes?

**Comment by avidrucker** [comment]
>  with passion (limiting beliefs are a self-fulfilling prophecy) , persistence (willingness to experience discomfort welcomes change), and grit (willingness to make mistakes and fail forward)

What are effective ways of fostering passion, persistence, and grit? Are these things measurable, and if so, are these measurements reliable and can they inform restorative action?

**Comment by seekanddefine** [comment]
What do you think about these claims? From [this article on Deloitte's Agile Tube Map](https://medium.com/tech-sojourna/7-things-wrong-with-deloittes-agile-tube-map-641192e20068)

-Individuals and interactions over processes and tools
-Working software over comprehensive documentation
-Customer collaboration over contract negotiation
-Responding to change over following a plan
